### PR TITLE
show followed and followers

### PR DESCRIPTION
**Pull request by candelarialopezz**

### CODE REVIEW

File: app/views/users/registrations/edit.html.haml [hunk 1]
```diff
@@ -30,6 +30,10 @@
       = f.password_field :current_password, autocomplete: 'current-password', class: 'form-control mb-2'
       - if resource.errors[:current_password].include?("can't be blank")
         = render 'errors/error_fields', errors: ['enter your password to confirm changes']
+      - elsif resource.errors[:current_password].include?("is invalid")
```
Comment (daizuoheng) [inline app/views/users/registrations/edit.html.haml:33]:
```suggestion
      - elsif resource.errors[:current_password].include?('is invalid')
```

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -15,13 +15,11 @@ def index
 
   def following
     @title = 'Following'
```
Comment (EnzoCozza) [inline app/controllers/users_controller.rb:17]: You are not using this variable, let's remove it!

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -15,13 +15,11 @@ def index
...
-    @user  = User.find(params[:id])
-    render 'following'
+    @user = User.find(params[:id])
   end
 
   def followers
     @title = 'Followers'
```
Comment (EnzoCozza) [inline app/controllers/users_controller.rb:22]: Same here

File: app/views/users/_following-followed.html.haml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+.followe-container
+  = link_to following_user_path(user.id), class: 'link_to_following' do
+    following
+    %br
+    = @user.following.count
```
Comment (EnzoCozza) [inline app/views/users/_following-followed.html.haml:5]: Let's use the local variable
```suggestion
    = user.following.count
```

File: app/views/users/followers.html.haml [hunk 5]
```diff
@@ -1,2 +1,15 @@
-followers
-= @user.followers.count
+.wrapper
+  .side
+    = render partial: 'side_user_data', locals: { user: @user, posts_count: @user.posts.count}
+    = render partial: 'following-followed', locals: { user: @user }
+    -if @user != current_user
```
Comment (EnzoCozza) [inline app/views/users/followers.html.haml:5]:
```suggestion
    - if @user != current_user
```

File: app/views/users/followers.html.haml [hunk 6]
```diff
@@ -1,2 +1,15 @@
-followers
-= @user.followers.count
+.wrapper
+  .side
+    = render partial: 'side_user_data', locals: { user: @user, posts_count: @user.posts.count}
```
Comment (EnzoCozza) [inline app/views/users/followers.html.haml:3]:
```suggestion
    = render partial: 'side_user_data', locals: { user: @user, posts_count: @user.posts.count }
```

File: app/views/users/following.html.haml [hunk 7]
```diff
@@ -1,2 +1,15 @@
-followers
-= @user.following.count
+.wrapper
+  .side
+    = render partial: 'side_user_data', locals: { user: @user, posts_count: @user.posts.count}
+    = render partial: 'following-followed', locals: { user: @user, following: true }
+    -if @user != current_user
```
Comment (EnzoCozza) [inline app/views/users/following.html.haml:5]:
```suggestion
    - if @user != current_user
```

File: app/views/users/following.html.haml [hunk 8]
```diff
@@ -1,2 +1,15 @@
-followers
-= @user.following.count
+.wrapper
+  .side
+    = render partial: 'side_user_data', locals: { user: @user, posts_count: @user.posts.count}
```
Comment (EnzoCozza) [inline app/views/users/following.html.haml:3]:
```suggestion
    = render partial: 'side_user_data', locals: { user: @user, posts_count: @user.posts.count }
```

File: app/views/users/show.html.haml [hunk 9]
```diff
@@ -1,7 +1,8 @@
 .wrapper
   .side
-    = render partial: 'side_user_data', locals: { user: @user }
-    - if @user != current_user
+    = render partial: 'side_user_data', locals: { user: @user, posts_count: @posts.count }
+    = render partial: 'following-followed', locals: { user: @user }
+    -if @user != current_user
```
Comment (EnzoCozza) [inline app/views/users/show.html.haml:5]:
```suggestion
    - if @user != current_user
```

File: app/controllers/users_controller.rb
Comment (EnzoCozza) [inline app/controllers/users_controller.rb:1]: Let's do a before_action for `following` and `followeres` to load @user

**Review by EnzoCozza — APPROVED** [review]

**Review by daizuoheng — APPROVED** [review]

**Pull request merged**